docs/Reference/ViewManipulation.md

```diff
@@ -126,7 +134,7 @@ const client = new jsc8({url: "https://gdn1.macrometa.io", token: "XXXX"});
 //---- OR ----
 const client = new jsc8({url: "https://gdn1.macrometa.io", apiKey: "XXXX"});
 
-const data = await client.createView("some-view", { links: { "some-collection": { } } });
+const data = await client.createView("some-view", { { "some-collection": { } } ,[ {field: "some-field", asc: true  } ]);
```

Comment:
this should be only `{ "some-collection": { }}`. there is one extra `{`. also for primarySort payload check indentation, two spaces after `true` and no space before `field`